internal/kafka/command_link_describe.go

```diff
@@ -45,22 +67,76 @@ func (c *linkCommand) describe(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	link, err := kafkaREST.CloudClient.GetKafkaLink(linkName)
+	cloudClient := kafkaREST.CloudClient
+	apiContext := context.WithValue(context.Background(), kafkarestv3.ContextAccessToken, cloudClient.AuthToken)
+	req := cloudClient.ClusterLinkingV3Api.GetKafkaLink(apiContext, cloudClient.ClusterId, linkName)
+	req = req.IncludeTasks(true)
+	res, httpResp, err := req.Execute()
+	link, err := res, kafkarest.NewError(cloudClient.GetUrl(), err, httpResp)
 	if err != nil {
 		return err
 	}
 
 	table := output.NewTable(cmd)
-	table.Add(newDescribeLink(link, ""))
-	table.Filter(getListFields(false))
-	return table.Print()
+	describeOut := newDescribeLink(link, "")
+	table.Add(describeOut)
+	isSerialized := output.GetFormat(cmd).IsSerialized()
+	if isSerialized {
+		// If we are serializing the output then there's no need to do any customization of the output. It will get
+		// correctly serialized.
+		table.Filter(getDescribeClusterLinksFields(true))
+		return table.Print()
+	} else {
+		// If we are not serializing the output, which means it's for human consumption, then we do some customization
+		// so it's more readable.
+		// The main link info gets output in table format which means it has two columns. Because there are multiple
+		// tasks, and each task itself can have multiple errors it's awkward to shove all the output into a single
+		// column. As a result we print a separate list dedicated to the tasks after the first table that contains the
+		// main link information.
+		table.Filter(getDescribeClusterLinksFields(false))
+		if err != nil {
+			return err
+		}
+		err = table.Print()
+		if err != nil {
+			return err
+		}
+		taskOuts := describeOut.Tasks
+		if len(taskOuts) > 1 {
+			list := output.NewList(cmd)
+			for i := range taskOuts {
+				t := taskOuts[i]
+				var errsStr bytes.Buffer
+				for i := range t.Errors {
+					eo := t.Errors[i]
+					errsStr.WriteString("Error code: ")
+					errsStr.WriteString(eo.ErrorCode)
+					errsStr.WriteString(" ")
+					errsStr.WriteString("Error message: ")
+					errsStr.WriteString(eo.ErrorMessage)
+					if i < len(t.Errors)-1 {
+						errsStr.WriteString(",")
+					}
+				}
+				list.Add(&humanTaskOut{
+					TaskName: t.TaskName,
+					State:    t.State,
+					Errors:   errsStr.String(),
+				})
+			}
+			return list.Print()
+		} else {
+			return nil
+		}
```

Comment:
We have six layers of indentation in the innermost block. Let's try to decrease that. Here's one idea but you could also put that string building code in a separate function too.
```suggestion
		if len(taskOuts) == 0 {
			return nil
		}
		
		list := output.NewList(cmd)
		for i := range taskOuts {
			t := taskOuts[i]
			var errsStr bytes.Buffer
			for i := range t.Errors {
				eo := t.Errors[i]
				errsStr.WriteString("Error code: ")
				errsStr.WriteString(eo.ErrorCode)
				errsStr.WriteString(" ")
				errsStr.WriteString("Error message: ")
				errsStr.WriteString(eo.ErrorMessage)
				if i < len(t.Errors)-1 {
					errsStr.WriteString(",")
				}
			}
			list.Add(&humanTaskOut{
				TaskName: t.TaskName,
				State:    t.State,
				Errors:   errsStr.String(),
			})
		}
		
		return list.Print()
```